README.md

```diff
@@ -144,12 +144,10 @@ cmake -S . -B build -DBUILD_SHARED_LIBS=OFF -DAVIF_CODEC_AOM=LOCAL -DAVIF_LIBYUV
 cmake --build build --parallel
 ```
 
-## Prebuilt Library (Windows)
+## Prebuilt Static Binaries (Windows)
```

Comment:
Nit: I suggest removing the word "Static".

It's not clear what "Static" means here. I think it means "statically-linked", i.e., the executables are linked with the libavif static library.

Reply:
Done in https://github.com/AOMediaCodec/libavif/pull/2462.

Reply:
Also done in https://github.com/AOMediaCodec/libavif/pull/2463.